packages/api/src/command/medical/patient/delete-patient.ts

```diff
@@ -19,43 +18,36 @@ export type DeleteOptions = {
   allEnvs?: boolean;
 };
 
-export const deletePatient = async (
-  patientDelete: PatientDeleteCmd,
-  options: DeleteOptions = {}
-): Promise<void> => {
+export const deletePatient = async (patientDelete: PatientDeleteCmd): Promise<void> => {
   const { id, cxId, facilityId: facilityIdParam, eTag } = patientDelete;
 
   const patient = await getPatientOrFail({ id, cxId });
   validateVersionForUpdate(patient, eTag);
 
   const facilityId = getFacilityIdOrFail(patient, facilityIdParam);
-
-  if (options.allEnvs || Config.isSandbox()) {
-    const fhirApi = makeFhirApi(cxId);
-
-    try {
-      // These need to run before the Patient is deleted (need patient data from the DB)
-      await Promise.all([
-        cwCommands.patient.remove(patient, facilityId).catch(err => {
-          if (err.response?.status !== 404) throw err;
-          console.log(`Patient not found @ CW when deleting ${patient.id} , continuing...`);
-          processAsyncError(deleteContext);
-        }),
-        fhirApi.deleteResource("Patient", patient.id).catch(processAsyncError(deleteContext)),
-        cqCommands.patient.remove(patient).catch(processAsyncError(deleteContext)),
-      ]);
-      await patient.destroy();
-    } catch (error) {
-      capture.error("Failed to delete patient", {
-        extra: {
-          context: deleteContext,
-          patientId: patient.id,
-          facilityId,
-          options,
-          error,
-        },
-      });
-      throw error;
-    }
+  const fhirApi = makeFhirApi(cxId);
+
+  try {
+    // These need to run before the Patient is deleted (need patient data from the DB)
+    await Promise.all([
+      cwCommands.patient.remove(patient, facilityId).catch(err => {
+        if (err.response?.status !== 404) throw err;
+        console.log(`Patient not found @ CW when deleting ${patient.id} , continuing...`);
+        processAsyncError(deleteContext);
+      }),
```

Comment:
This code doesn't work properly. `processAsyncError` returns a function to be passed to `.catch()`.

I think the logic we wanted here was something like this:

```ts
          if (err.response?.status === 404) {
            console.log(`Patient not found @ CW when deleting ${patient.id} , continuing...`);
            return;
          }
          processAsyncError(deleteContext)(err);
```